README.md

```diff
@@ -74,7 +74,20 @@ $version = XdebugHandler::getSkippedVersion();
 ```
 
 ### Output
-The library is silent unless the `-vvv` command-line option is present, which results in the output of simple status messages.
+The library is silent unless the `-vvv` command-line option is present, which results in the output of status messages on stdout.
```

Comment:
Could you please remove the command line argument parsing completely? It belongs into the application, not a library. You could implement your Logger to parse `-vvv` if you want that in your application but this shouldn't be part of the package for every consumer.